server/jetstream_versioning.go

```diff
@@ -76,12 +98,41 @@ func setConsumerAssetVersionMetadata(cfg *ConsumerConfig, prevCfg *ConsumerConfi
 	cfg.Metadata[JSRequiredLevelMetadataKey] = requiredApiLevel
 }
 
+// includeDynamicConsumerAssetVersionMetadata adds dynamic fields into the (copied) metadata.
+func includeDynamicConsumerAssetVersionMetadata(cfg *ConsumerConfig) *ConsumerConfig {
```

Comment:
I prefer way shorter names in Go code and especially the server code base.

Reply:
Agree, mostly just ran with initial naming from the ADR, and added more words for dynamic :sweat_smile: .

Currently they are named:
- `includeDynamicConsumerAssetVersionMetadata` for dynamic metadata
- `setConsumerAssetVersionMetadata` for static metadata

Have shortened them to:
- `setDynamicConsumerMetadata` for dynamic metadata
- `setStaticConsumerMetadata` for static metadata (done in the parent PR instead of this one)

Do they need to be shortened further?